UM/Math/Color.py

```diff
@@ -67,3 +67,25 @@ def fromARGB(value):
             (value & 0x000000ff) >> 0,
             (value & 0xff000000) >> 24
         )
+
+    ##  Returns a new Color constructed from a 7-character string "#RRGGBB" format.
+    #
+    #   \param value A 7-character string representing a color in "#RRGGBB" format.
+    #   \return A Color constructed from the components of value.
+    @staticmethod
+    def fromRGBString(value):
```

Comment:
The thing is, UM.Math.Color is meant as an RGB_A_ color value. The API should reflect that, which is why I would like to make sure factory functions like this support the full class, not just a special case.